### PR TITLE
patch release 2023-05-06

### PR DESCRIPTION
**Pull request by Goncharo**
Ref: #290 

### Dependencies

- Downstream: tbd

### Description

patch release 2023-05-06

### Release Plan

- [x] release api 3.1.4
- [x] merge this
- [ ] merge downstream

### CODE REVIEW

**Review by Orta21 — APPROVED** [review]

**Review by Orta21 — APPROVED** [review]

File: api/app/src/providers/withings.ts [hunk 1]
```diff
@@ -105,16 +106,24 @@ export class Withings extends Provider implements OAuth2 {
...
+          {
+            headers: {
+              Authorization: `Bearer ${access_token}`,
+            },
+          }
+        );
+        console.log(resp.data);
+      } catch (error) {
+        capture.error(error, {
+          extra: { context: `withings.postAuth` },
```
Comment (leite08) [inline api/app/src/providers/withings.ts:122]: can we send the URL and the body as well?

**Review by leite08 — APPROVED** [review]
LGTM, just a nice to have to add more info about the issue on Withings report to Sentry

File: api/app/src/providers/withings.ts (hunk 1 above)
Comment (Goncharo) [reply api/app/src/providers/withings.ts:122]: There is no body, I can send the url though

Comment (Goncharo) [reply api/app/src/providers/withings.ts:122]: Nvm will send both

Comment (Goncharo) [reply api/app/src/providers/withings.ts]: nvm again - I think we were doing the sub incorrectly in the first place

**Review by Orta21 — APPROVED** [review]

**Pull request merged**